Source/WebCore/animation/AnimationTimelinesController.h

```diff
@@ -67,22 +67,20 @@ class AnimationTimelinesController final : public CanMakeCheckedPtr<AnimationTim
     WEBCORE_EXPORT void resumeAnimations();
     bool animationsAreSuspended() const { return m_isSuspended; }
 
-    void registerNamedScrollTimeline(const AtomString&, Element&, ScrollAxis);
-    void unregisterNamedScrollTimeline(const AtomString&);
-    ScrollTimeline* scrollTimelineForName(const AtomString&) const;
-
-    void registerNamedViewTimeline(const AtomString&, Element&, ScrollAxis, ViewTimelineInsets&&);
-    void unregisterNamedViewTimelineForSubject(const AtomString&, const Element&);
-    ViewTimeline* viewTimelineForNameAndSubject(const AtomString&, const Element&) const;
+    void registerNamedScrollTimeline(const AtomString&, const Element&, ScrollAxis);
+    void registerNamedViewTimeline(const AtomString&, const Element&, ScrollAxis, ViewTimelineInsets&&);
+    void unregisterNamedTimeline(const AtomString&, const Element&);
+    AnimationTimeline* timelineForName(const AtomString&, const Element&) const;
 
 private:
     ReducedResolutionSeconds liveCurrentTime() const;
     void cacheCurrentTime(ReducedResolutionSeconds);
     void maybeClearCachedCurrentTime();
 
+    Vector<Ref<AnimationTimeline>>& timelinesForName(const AtomString&);
+
+    UncheckedKeyHashMap<AtomString, Vector<Ref<AnimationTimeline>>> m_nameToTimelineMap;
```

Comment:
This should be a `Vector<Ref<ScrollTimeline>>` I believe, since we're not going to register any timeline that isn't a scroll timeline.

Reply:
Fixed.